README.md

```diff
@@ -51,9 +51,12 @@ Here are the [instrumentation
 libraries](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumentation-library):
 
 * [ASP.NET Core](./src/OpenTelemetry.Instrumentation.AspNetCore/README.md)
```

Comment:
Since we are explicitly listing the individual packages. Should we mention [Grpc.AspNetCore](https://www.nuget.org/packages/Grpc.AspNetCore) as well here.

Reply:
We could do that. I'll do that in another PR.